keepalive/keepalive.go

```diff
@@ -33,7 +33,19 @@ import (
 type ClientParameters struct {
 	// After a duration of this time if the client doesn't see any activity it
 	// pings the server to see if the transport is still alive.
-	// If set below 10s, a minimum value of 10s will be used instead.
+	// If set below 10s (including 0), a minimum value of 10s will be used
```

Comment:
The "including 0" is probably not needed if we get rid of the  comment below as mentioned.

Reply:
Done